tox.ini

```diff
@@ -30,6 +30,7 @@ deps =
     pytest41: pytest>=4.1,<4.2
     pytest41: attrs==17.4.0
     xdist: pytest-xdist>=1.15
+    xdist: pytest<5.3.4
```

Comment:
What about an extra factor `pytest353`?
We should still test xdist also with current pytest.

Reply:
Definitely. This is just temporary so I could see if it was hitting those lines when using pytest 5.3.3. I tried testing it locally, but it always passes for me, so I think something is off between my machine and whatever is going on in Travis and I wanted to get that resolved as well